html-css/week1/index.html

```diff
@@ -0,0 +1,110 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css" crossorigin="anonymous">
+    <title>Elnaz Talebi portfolio web page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="nav-container">
+    <nav class="navbar">
+      <ul class="nav-list">
+        <li>
+          <a href="#welcome-section">About</a>
+        </li>
+        <li>
+          <a href="#projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#contacts">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    </div>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Hey, I am Elnaz</h1>
+      <br>
+      <p>This page is about various personal projects</p>
+    </section>
+    <section id="projects" class="projects">
+      <h2 class="projects-section-header">These are some of my projects</h2>
+      <div class="project-grid">
+        <a href="https://codepen.io/Elnaz-Talebi/pen/XJrqNOE" target="_blank" class="project-box">
+          <img class="project-image" src="https://raw.githubusercontent.com/Elnaz-Talebi/Portfolio/refs/heads/main/Images/Survey-form-picture.JPG" alt="project">
+          <p>Survey Form</p>
+        </a>
+        <a href="https://codepen.io/Elnaz-Talebi/full/MWNXJEy" target="_blank" class="project-box">
+          <img class="project-image" src="https://raw.githubusercontent.com/Elnaz-Talebi/Portfolio/refs/heads/main/Images/first-project-picture.JPG" alt="project">
+          <p>Event by Hack your future</p>
+        </a>
+        <a href="https://codepen.io/Elnaz-Talebi/full/MWNXJEy" target="_blank" class="project-box">
+          <img class="project-image" src="https://raw.githubusercontent.com/Elnaz-Talebi/Portfolio/refs/heads/main/Images/first-project-picture.JPG" alt="project">
+          <p>Event by Hack your future</p>
+        </a>
+        <a href="https://codepen.io/Elnaz-Talebi/full/MWNXJEy" target="_blank" class="project-box">
+          <img class="project-image" src="https://raw.githubusercontent.com/Elnaz-Talebi/Portfolio/refs/heads/main/Images/first-project-picture.JPG" alt="project">
+          <p>Event by Hack your future</p>
+        </a>
+        <a href="https://codepen.io/Elnaz-Talebi/full/MWNXJEy" target="_blank" class="project-box">
+          <img class="project-image" src="https://raw.githubusercontent.com/Elnaz-Talebi/Portfolio/refs/heads/main/Images/first-project-picture.JPG" alt="project">
+          <p>Event by Hack your future</p>
+        </a>
+        <a href="https://codepen.io/Elnaz-Talebi/full/MWNXJEy" target="_blank" class="project-box">
+          <img class="project-image" src="https://raw.githubusercontent.com/Elnaz-Talebi/Portfolio/refs/heads/main/Images/first-project-picture.JPG" alt="project">
+          <p>Event by Hack your future</p>
+        </a>
```

Comment:
In this case, I would recommend to remove `p` tag and writing down the content in `p` tag directly to `a` tag instead 
```
<a href="https://codepen.io/Elnaz-Talebi/full/MWNXJEy" target="_blank" class="project-box">
   Event by Hack your future
 <img class="project-image" src="https://raw.githubusercontent.com/Elnaz- Talebi/Portfolio/refs/heads/main/Images/first-project-picture.JPG" alt="project"> 
</a>
```

Please edit all of them ☺️